src/test/java/org/sonatype/nexus/plugins/conda/internal/util/CondaPathUtilsTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ * Sonatype Nexus (TM) Open Source Version
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * All rights reserved. Includes the third-party code listed at http://links.sonatype.com/products/nexus/oss/attributions.
+ *
+ * This program and the accompanying materials are made available under the terms of the Eclipse Public License Version 1.0,
+ * which accompanies this distribution and is available at http://www.eclipse.org/legal/epl-v10.html.
+ *
+ * Sonatype Nexus (TM) Professional Version is available from Sonatype, Inc. "Sonatype" and "Sonatype Nexus" are trademarks
+ * of Sonatype, Inc. Apache Maven is a trademark of the Apache Software Foundation. M2eclipse is a trademark of the
+ * Eclipse Foundation. All other trademarks are the property of their respective owners.
+ */
+package org.sonatype.nexus.plugins.conda.internal.util;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.sonatype.goodies.testsupport.TestSupport;
+import org.sonatype.nexus.repository.view.matchers.token.TokenMatcher;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.Mock;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.is;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.mockito.Mockito.when;
+
+public class CondaPathUtilsTest
+    extends TestSupport
+{
+  private CondaPathUtils underTest;
+
+  @Mock
+  TokenMatcher.State state;
+
+  private Map<String, String> tokens;
+
+  @Before
+  public void setUp() throws Exception {
+    underTest = new CondaPathUtils();
+    tokens = setupTokens("3.0.0", "imaginary/path", "123", "numpy", "osx");
+  }
+
+  @Test
+  public void arch() throws Exception {
+    when(state.getTokens()).thenReturn(tokens);
```

Comment:
super minor: Any reason not to move `when(state.getTokens()).thenReturn(tokens);` into the `setUp()` method?

Reply:
No huge reason, but I like scoping responses of mocks within the actual test, to kinda counter global behavior (if you wanted to write some other tests blah blah blah)

Reply:
In: f048184